.github/workflows/build-and-test.yml

```diff
@@ -24,9 +24,9 @@ jobs:
     - name: Test
       run: |
         cd ./tests/WaySON.UnitTests/
-        dotnet test --no-restore /p:CollectCoverage=true /p:CoverletOutput=/home/runner/work/wayfair-text-json/wayfair-text-json/ /p:CoverletOutputFormat=opencover
+        dotnet test --no-restore /p:CollectCoverage=true /p:CoverletOutput=/home/runner/work/wayson/wayson/ /p:CoverletOutputFormat=opencover

```

Comment:
For URLs it won't be an issue, but casing will matter for file paths. The repo is WaySON, so I expect that's how it will be checked out.

Reply:
Good catch. Let's see how this run goes and I'll update if needed. 👍

Reply:
Looks like the code coverage upload failed - just pushed a fix for these paths.